uchileedxlogin/views.py

```diff
@@ -559,6 +559,8 @@ def login_user(self, request, username):
         user_data = self.get_user_data(username)
         user_data['username'] = username
         edxlogin_user = self.get_or_create_user(user_data)
+        edxlogin_user.have_sso = True
+        edxlogin_user.save()
```

Comment:
Puedes agregar un if not edxlogin_user.have_sso, antes de esto, para no guardar cada vez que hagan login